docs/references/config/config.toml.md

```diff
@@ -1205,6 +1205,20 @@ might become invalid. Setting `recheck = true` will go through the remaining tra
 If your application may remove transactions passed by CometBFT to your `PrepareProposal` handler,
 you probably want to set this configuration to `true` to avoid possible leaks in your mempool
 (transactions staying in the mempool until the node is next restarted).
+
+### mempool.recheck_timeout
+Time to wait for the application to return CheckTx responses after the rechecking process was
+started. Responses that arrive after the timeout expires are discarded.
+```toml
+recheck_timeout = "300ms"
```

Comment:
As mentioned above, I find 300ms too low for advanced apps (which have to verify signatures, etc)

Reply:
Yes, I thought the same. I changed to 1 second. I think more than that will be too much, though it greatly depends on the application.